doc/manual/43-backup-commands.en.md

```diff
@@ -339,6 +339,34 @@ the `--recovery-staging-path` option with the `barman recover` command. If
 you do neither of these things and attempt to recover a compressed backup
 then Barman will fail rather than try to guess a suitable location.
 
+### Recovering incremental backups
+
+If a backup is incremental, `barman recover` is able to combine the
+backup on recovery using `pg_combinebackup` and the chain of backups.
+A chain of backup is the tree branch that goes from the full backup
+to the one requested for the recovery. This is a multi-step process:
+
+1. The chain of backups is combined into a new synthetic backup in a
+   staging directory on the local server using `pg_combinebackup`. For
+   any type of recover (local or remote), the synthetic backup is created
+   locally in the barman server.
+2. If it's a remote recover, the content is copied to the final destination
+   using Rsync. Otherwise, when it's a local recover, the content is just
+   moved to the final destination.
+3. The staging directory for the backup is removed.
```

Comment:
I wonder if users could think the path pointed by `--local-staging-path` is removed?
I mean, we remove the folder of the synthetic backup, which is created inside `--local-staging-path`, not the `--local-staging-path` itself.

Thoughts?

Reply:
I think that part means only the staging folder for the backup! `the staging directory FOR THE BACKUP is removed.`  

But it is prone to errors, yes!